src/components/views/UserHand.js

```diff
@@ -0,0 +1,139 @@
+import React, {useEffect, useState} from 'react';
+import PropTypes from "prop-types";
+import {useHistory} from "react-router-dom";
+import {api, handleError} from "../../helpers/api";
+import CardButton from "../ui/CardButton";
+import "styles/ui/CardButton.scss";
+import "styles/views/UserHand.scss";
+
+const UserHand = () => {
+    const [blackCard, setBlackCard] = useState(null)
+    const [whiteCards, setWhiteCards] = useState(null)
+    const userId = localStorage.getItem("id")
+    const token = localStorage.getItem("token")
+    const cardsPerHand = 8 //constant of how many cards there are in a users hand
+    const history = useHistory();
+
+    const WhiteCard = ({card}) => {
+        return(
+            <CardButton className={"card whiteCard simple"}
+                        children={card.text}
+                        key={card.id}
+            />
+        );
+    };
+    WhiteCard.propTypes = {
+        card: PropTypes.object
+    };
+
+    useEffect( () => {
+        async function getBlackCard() {
+            try {
+                const response = await api.get(`/users/${userId}/games/blackCards/current`,
+                    {
+                        headers: {
+                            "authorization": token
+                        }
+                    });
```

Comment:
You can remove that piece of code. We have an interceptor that inserts the header automatically